src/js/operations/Geo.js

```diff
@@ -0,0 +1,33 @@
+L.DNC.Geo = L.Class.extend({
```

Comment:
Why the switch from `TurfOperations` to `Geo`? Do you predict that this project will always be a wrapper for `turf.js`? I had envisioned other operations not necessarily covered within `turf.js`, such as generating heatmaps.

This may be a premature concern, but I think it'd be wise to keep open the door for expanding beyond `turf.js`.